datastore/common/objects/Vrf.test.cpp

```diff
@@ -74,6 +74,6 @@ BOOST_AUTO_TEST_CASE(testSetters)
 
   BOOST_CHECK_EQUAL(id, vrf.getId());
   BOOST_CHECK_EQUAL(2, vrf.getIfaces().size());
-  BOOST_CHECK_EQUAL("eth0", vrf.getIfaces().at(0));
-  BOOST_CHECK_EQUAL("eth1", vrf.getIfaces().at(1));
+  BOOST_CHECK_EQUAL("eth0", vrf.getIfaces().at(0).getName());
+  BOOST_CHECK_EQUAL("eth1", vrf.getIfaces().at(1).getName());
```

Comment:
Change this to potentially catch if someone changes the logic to incorporate interface name expansion.  Have to change the below and above (~73).
```suggestion
  BOOST_CHECK_EQUAL("gi0", vrf.getIfaces().at(1).getName());
```